src/libraries/System.IO.Compression/ref/System.IO.Compression.cs

```diff
@@ -51,7 +51,7 @@ public override void SetLength(long value) { }
         public override void Write(byte[] buffer, int offset, int count) { }
         public override void Write(System.ReadOnlySpan<byte> buffer) { }
         public override System.Threading.Tasks.Task WriteAsync(byte[] buffer, int offset, int count, System.Threading.CancellationToken cancellationToken) { throw null; }
-        public override System.Threading.Tasks.ValueTask WriteAsync(System.ReadOnlyMemory<byte> buffer, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public override System.Threading.Tasks.ValueTask WriteAsync(System.ReadOnlyMemory<byte> buffer, System.Threading.CancellationToken cancellationToken) { throw null; }
```

Comment:
This will be a source-breaking change.  Can we double check which is intended here?

Reply:
Pretty sure this is a bug in the src rather than the ref.  I see the base `WriteAsync` has the default parameter https://github.com/dotnet/runtime/blob/4da708a7060d238354d79823e58839c1d92f7fd1/src/libraries/System.Runtime/ref/System.Runtime.cs#L10773

Reply:
I was just about to paste this, you're fast. Yes, the src is wrong.

Reply:
But as you said, this is a breaking change. I guess we'll have to take it anyway.

Reply:
Fixed the ref.